### DISCUSSION TITLE
CreateQuery , DocumentQuery , QueryFindOneAndUpdateOptions

### DISCUSSION THREAD

**Discussion by kamalkech** [Automattic/mongoose#12648] [Q&A]
i am using "mongoose": "~6.6.0" but not export CreateQuery , DocumentQuery , QueryFindOneAndUpdateOptions
so in this version are u replace those function to others ??

**Comment by vkarpov15** [comment]
Can you please provide context for how you're using these types and what version you're upgrading from?
